graph/Cargo.toml

```diff
@@ -13,7 +13,7 @@ diesel = { version = "1.3.2", features = ["postgres", "serde_json", "numeric", "
 # But, graph-patches also contains Jannis' https://github.com/paritytech/ethabi/pull/140 PR
 # to ethabi. Which we believe will not get merged. For now, we shall deviate from ethabi,
 # but long term we want to avoid forking off ethabi if possible
-ethabi = { git = "https://github.com/graphprotocol/ethabi.git", branch = "graph-patches" }
+ethabi = { git = "https://github.com/graphprotocol/ethabi.git", branch = "ford/tuple" }
```

Comment:
We'll have to take this out of the PR and run `cargo update graph-patches` instead once ethabi is merged. That will update the ethabi commit in `Cargo.lock`.